addons/laserpointer/XEH_preInit.sqf

```diff
@@ -6,4 +6,11 @@ PREP_RECOMPILE_START;
 #include "XEH_PREP.hpp"
 PREP_RECOMPILE_END;
 
+["visionMode", {
```

Comment:
I was intending this to fully disable the draw EH. Because the if statement inside the draw EH is what probably costs performance.